lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -774,11 +774,16 @@ class MongoClientInterface {
                 return cb(errors.InternalError);
             }
             this.getLatestVersion(c, objName, params.vFormat, log, (err, mstObjVal) => {
+                if (err && err.is.NoSuchKey) {
```

Comment:
Now that I look at it we can rewrite this as ( I believe ):
```suggestion
                if (err?.is.NoSuchKey) {
```
Maybe a cleanup pass later